src/routes/notification.route.js

```diff
@@ -8,11 +8,12 @@ const app = express.Router();
 const { checkToken } = UserMiddleware;
 const { checkNotificationExists, checkNotificationOwner } = NotificationMiddleware;
 const {
-  getUserNotifications, updateNotificationStatus, getSingleNotification,
+  getUserNotifications, updateNotificationStatus, getSingleNotification, markAllNotificationsAsRead
 } = NotificationController;
 
 app.get('/', checkToken, getUserNotifications);
 app.get('/:id', checkToken, checkNotificationExists, checkNotificationOwner, getSingleNotification);
 app.patch('/:id/:status', checkToken, checkNotificationExists, checkNotificationOwner, updateNotificationStatus);
+app.patch('/all', checkToken, markAllNotificationsAsRead);
```

Comment:
add the middleware to check if the notifications belong to the user who logged in

Reply:
yes I did that because I am using userid from the token @alexismajya